UDS_Module2_D365Client/Program.cs

```diff
@@ -38,27 +39,26 @@ static void Main(string[] args)
                 rent.cr9d3_pickup_location = new OptionSetValue(new Random().Next(0, 2));
                 rent.cr9d3_return_location = new OptionSetValue(new Random().Next(0, 2));
 
-                if (!StatusCodeIsActive(i))
+                if (!StatusCodeIsActive(recordNumber))
                 {
                     rent.statecode = cr9d3_rentState.Inactive;
                 }
 
-                rent.statuscode = new OptionSetValue( RandomValues.GetRandomStatusReasonValue(i));
-                
-                var pickupReport = new cr9d3_cartransferreport();
-
-                pickupReport.cr9d3_car = rent.cr9d3_car;
-                pickupReport.cr9d3_date = pickupDate;
-                service.Create(pickupReport);
-
-                                
-                CrmRequests.GetPickupReport(service,carName, carValue, (DateTime)pickupDate); //need tests
-
-                //service.Create(rent);          
+                rent.statuscode = new OptionSetValue(RandomValues.GetRandomStatusReasonValue(recordNumber));
+                var statusReason = rent.statuscode;
 
+                if (statusReason.Value == 970300001) CreatePickUpReport(service, rent, recordNumber);
```

Comment:
could you please add curly braces for CreatePickUpReport(service, rent, recordNumber) and place them below if clause